appcell-resources/js/login.js

```diff
@@ -149,7 +150,10 @@ lg.setBizTheme = function() {
     if (cellType == "Organization") {
         $('body').addClass('body-biz');
         $('.login_btn').addClass('login_btn-biz');
-    }
+    } else {
+        $('body').addClass('body');
+    };
+    $('body > div').toggle();
```

Comment:
The routines are missing:
1. mode=global with invalid target cell URL
2. Tutorial